From: David Wronek <david@mainlining.org>
Date: Fri, 8 Sep 2023 21:54:20 +0300
Subject: FROMLIST: dt-bindings: phy: qcom,qusb2: Add the SM7150 compatible

Add devicetree compatible for the usb phy on SM7150 SoC.

Signed-off-by: David Wronek <david@mainlining.org>
Signed-off-by: Danila Tikhonov <danila@jiaxyga.com>
---
 Documentation/devicetree/bindings/phy/qcom,qusb2-phy.yaml | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Documentation/devicetree/bindings/phy/qcom,qusb2-phy.yaml b/Documentation/devicetree/bindings/phy/qcom,qusb2-phy.yaml
index 39851ba..51dd79e 100644
--- a/Documentation/devicetree/bindings/phy/qcom,qusb2-phy.yaml
+++ b/Documentation/devicetree/bindings/phy/qcom,qusb2-phy.yaml
@@ -36,6 +36,7 @@ properties:
               - qcom,sdm670-qusb2-phy
               - qcom,sdm845-qusb2-phy
               - qcom,sm6350-qusb2-phy
+              - qcom,sm7150-qusb2-phy
           - const: qcom,qusb2-v2-phy
   reg:
     maxItems: 1
